pnpm-lock.yaml

```diff
@@ -8,6 +8,12 @@ overrides:
   mime: ^3.0.0
 
 dependencies:
+  '@casl/ability':
+    specifier: ^6.3.3
+    version: 6.5.0
+  '@casl/vue':
+    specifier: ^2.2.1
+    version: 2.2.1(@casl/ability@6.5.0)(vue@3.3.4)
```

Comment:
Some dependencies (in package.json) are not in the lock file entries. so we need this updated lock file. CI wants to install the dependencies using lock-file which is a good practice.